section1/exercises/variables.rb

```diff
@@ -11,49 +11,56 @@
 # Write code that saves the string 'Dobby' to a variable and
 # prints what that variable holds to the terminal:
 house_elf = "Dobby"
-# YOUR CODE HERE
+puts house_elf
 
 # Write code that saves the string 'Harry Potter must not return to Hogwarts!'
 # and prints what that variable holds to the terminal:
-# YOUR CODE HERE
+puts "#{name} must not return to Hogwarts!"
 
 # Write code that adds 2 to the `students` variable and
 # prints the result:
 students = 22
-# YOUR CODE HERE
+students = 22 + 2
```

Comment:
✅

Reply:
Though be aware you've just rewritten the original `students` variable that was 22, and now `students` can only be 24